### PR TITLE
add 'lab' to scenario tests

### PR DESCRIPTION
**Pull request by dnalborczyk**
we should probably also add `lab` to the README.

the only way I know how to load it is `node -r esm ./node_modules/.bin/lab my-tests`

I suppose the `--require` option would need to be implemented by `lab`.

### CODE REVIEW

**Comment by jdalton** [comment]
> the only way I know how to load it is `node -r esm ./node_modules/.bin/lab my-tests`

We should suggest `npx -n='-r esm' lab my-tests`

> I suppose the `--require` option would need to be implemented by lab.

That would be an awesome route and would enable other loaders like `@babel/register`.

**Comment by jdalton** [comment]
The reason you're seeing fails on Windows is because you're trying to execute lab from the `.bin` folder using the *nix branch. For Windows there is a `.cmd` file in there to call. That said you can avoid this by calling the `lab/bin/lab` as the path. Similar to the `jestPath`:
```js
const jestPath = path.resolve("../node_modules/jest/bin/jest.js")
```
so
```js
const labPath = path.resolve("../node_modules/lab/bin/lab")
```

File: test/scenario-tests.mjs [hunk 1]
```diff
@@ -300,6 +303,38 @@ describe("scenario tests", function () {
     })
   })
 
+  describe("should work with lab", function () {
+
```
Comment (jdalton) [inline test/scenario-tests.mjs]: Nit: No newline here

**Comment by dnalborczyk** [comment]
> The reason you're seeing fails on Windows is because you're trying to execute lab from the .bin folder using the *nix branch. For Windows there is a .cmd file in there to call. That said you can avoid this by calling the lab/bin/lab as the path. Similar to the jestPath:
> 
> const jestPath = path.resolve("../node_modules/jest/bin/jest.js")
> so
> 
> const labPath = path.resolve("../node_modules/lab/bin/lab")

didn't know that. thank you!

**Pull request merged**